packages/compass-utils/src/ipc.ts

```diff
@@ -1,17 +1,37 @@
 import { BrowserWindow, ipcMain, ipcRenderer } from 'electron';
+import { serialize } from 'v8';
```

Comment:
Just a side note, but this is yet another good indicator that it was indeed not a great idea to have the IPC code in this package, compass-utils was never intended to have hard dependencies on platform-specific code, it was really just about small utilities that don’t have a better place in our monorepo and aren’t quite complex enough to warrant a package of their own. (That’s also why `electron`/`@electron/remote` are optional dependencies in the package.json.)

Let’s do a follow-up PR to move it into its own package? Happy to do that myself as well.

Reply:
We can move it to hadron-ipc I guess, it's anyway almost exactly what we want from that package, just with an interface that uses more modern electron features

Reply:
I'll leave this for a follow-up